testsuite/tests/runtime-errors/Makefile

```diff
@@ -30,8 +30,9 @@ compile:
 	    $(OCAMLOPT) -w a -o $$F.native$(EXE) $$f; \
 	  fi; \
 	done
+	$(if $(findstring win32,$(UNIX_OR_WIN32)),:, \
```

Comment:
no, `HAS_STACK_OVERFLOW_DETECTION` enables the very Unix-specific `SIGSEGV` handling code that handles the stack overflow in `signals_asm.c` ; that just won't compile on Win32.